kuma/api/v1/views.py

```diff
@@ -287,6 +288,7 @@ class APISearchView(SearchView):
 
 @waffle_flag('bc-signals')
 @api_view(['POST'])
+@throttle_classes([UserRateThrottle])
```

Comment:
I don't see where you've set the `rate` limit for the `UserRateThrottle` class, so my understanding is that this will do nothing. You'll either need to do something like this:
```python
class OncePerDayUserThrottle(UserRateThrottle):
    rate = '1/day'
...
@throttle_classes([OncePerDayUserThrottle])
```
or instead of `throttle_classes`, just use the rate-limit decorator we use everywhere else in the kuma code:
```python
@ratelimit(key='user_or_ip', rate='1/d', block=True)
```

Reply:
Isn't it too much to throttle to ones per day?

Reply:
> Isn't it too much to throttle to ones per day?

yes, we should start with 10/day, and can adjust if necessary